### ISSUE TITLE
Device share link - causes Server restart 

### ISSUE THREAD

**Issue by VINISHVV** [Ylianst/MeshCentral#3292]
Dear @Ylianst,

While connecting the system through the share link ( Unlimite )the server gets restarted. 

Meshcentral Version - 0.9.51 

Error logs show below error messages.

```
c:\MSAT\node_modules\meshcentral\meshdesktopmultiplex.js:1103
                if (userid == null) { rights = MESHRIGHT_REMOTECONTROL; } else { rights = parent.GetNodeRights(user, routing.meshid, command.nodeid); }
                                                                                                                             ^
TypeError: Cannot read property 'meshid' of undefined
    at Object.CreateMeshRelayEx2.obj.sendAgentMessage (c:\MSAT\node_modules\meshcentral\meshdesktopmultiplex.js:1103:126)
    at Array.<anonymous> (c:\MSAT\node_modules\meshcentral\meshdesktopmultiplex.js:1331:25)
    at fileBulkReadCompleted (c:\MSAT\node_modules\meshcentral\db.js:2082:106)
    at c:\MSAT\node_modules\mongodb\lib\utils.js:530:9
    at c:\MSAT\node_modules\mongodb\lib\cursor\abstract_cursor.js:244:32
    at c:\MSAT\node_modules\mongodb\lib\cursor\abstract_cursor.js:484:55
    at c:\MSAT\node_modules\mongodb\lib\utils.js:530:9
    at c:\MSAT\node_modules\mongodb\lib\sessions.js:136:24
    at Object.maybePromise (c:\MSAT\node_modules\mongodb\lib\utils.js:516:5)
    at ClientSession.endSession (c:\MSAT\node_modules\mongodb\lib\sessions.js:133:24)
```

**Comment by Ylianst** [comment]
Thanks, working on it now!

**Comment by Ylianst** [comment]
Just published MeshCentral v0.9.52 with a fix for this, let me know if it works. Thank you for reporting!

**Comment by VINISHVV** [comment]
Dear Ylian,

it's fixed. Thanks for the quick response.
